### DISCUSSION TITLE
Is it possible to customize the (Purchase Order) CSV exports?

### DISCUSSION THREAD

**Discussion by Guusggg** [inventree/InvenTree#4267] [Q&A] [answered]
Hi,

I know there's a way to change the reports by editing a template. I was wondering if CSV functions the same way? Currently there's all kind of useful information in the export of a Purchase Order CSV, except for the part number, which we dearly need.

I've looked around in the files a bit, and found nothing. Nothing in the settings either. I don't mind editting the source of my instance, however I would like to try to find another way if possible.

Thank you for reading!

**Comment by SchrodingersGat** [comment, marked as answer]
Hey @Guusggg - currently there are not many options to customize CSV data export. It would require additional code (and someone with time willing to commit to this :) )

**Reply by Guusggg** [reply]
Not a problem at all, thank you for the reply! I might look into it but seeing as the reporting feature is already very complete, it might not be needed.
